src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -52,8 +55,31 @@
  */
 public class ManualCondition extends PromotionCondition {
     private String users;
+        
     private List<ParameterDefinition> parameterDefinitions = new ArrayList<ParameterDefinition>();
     
+    
+    //Code to fetch Approve Promotion Button label from properties file
+    private String executeButtonLabel;
+    
+    public String getExecuteButtonLabel() {
+    	try {
+    		//to use Custom label, add deamon property 'PROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL' to "/etc/init.d/jenkins" file in value to JAVA_CMD
+    		//sample: JAVA_CMD="$JENKINS_JAVA_CMD $JENKINS_JAVA_OPTIONS -DJENKINS_HOME=$JENKINS_HOME -DPROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL=Execute -jar $JENKINS_WAR"
+    		executeButtonLabel = System.getProperty("PROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL");
+    		if(executeButtonLabel == null) {
+    			executeButtonLabel = "Approve";	//if property is not set in "/etc/init.d/jenkins" default value is "Approve"
+    		}
+    		
+    	} catch (Exception e) {
+    		e.printStackTrace();
+    		executeButtonLabel = "Approve";
```

Comment:
🐛 You converted a localizable string to a non-localizable one. See https://wiki.jenkins-ci.org/display/JENKINS/Internationalization